### ISSUE TITLE
.Net Core 3.0 libraries from 7.1 will install .net core 2.2 in the project

### ISSUE THREAD

**Issue by jurjen74** [PiranhaCMS/piranha.core#932]
When I create an empty .Net Core 3.0 MVC WebApplication and add Piranha.Manager 7.1.0 from nuget, .NET Core 2.2 is also installed in type project, leading to loads of type problems:

![image](https://user-images.githubusercontent.com/6205945/69901137-10fc5200-137e-11ea-929b-7390c013a029.png)

Adding a project reference to the source from github does work.

From the nuspec:

```
        <dependency id="Microsoft.AspNetCore" version="2.2.0" />
        <dependency id="Microsoft.AspNetCore.Mvc" version="2.2.0" />
        <dependency id="Microsoft.AspNetCore.Session" version="2.2.0" />
        <dependency id="Microsoft.AspNetCore.SignalR" version="1.1.0" />
        <dependency id="Microsoft.AspNetCore.StaticFiles" version="2.2.0" />
        <dependency id="Microsoft.Extensions.FileProviders.Embedded" version="2.2.0" />
```

**Comment by mikaellindemann** [comment]
AFAIK the published packages are still for AspNetCore 2.2 while the master branch of the repo is transitioning to Piranha 8.0 which is going to support AspNetCore 3.0.

**Comment by tidyui** [comment]
The answer @mikaellindemann gave is correct. The master branch is our development branch which will be released as 8.0 BUT it will be compiled for .NET Core 3.1 as this will be LTS. 7.1 is the last version we’ll release for 2.2. We don’t see any point in releasing a version for .NET Core 3.0 as this is only a short term release
